src/CardsDeck/CardsDriver.cpp

```diff
@@ -28,11 +36,11 @@ void testCards() {
     cout << "Card 4 type: " << card4.getType() << "\n";
     cout << "Card 5 type: " << card5.getType() << "\n" << "\n";
     cout << "...Setting Card5 to a valid type..." << "\n";
-    card1.setType("Diplomacy"); // testing setter; changes invalid type to something valid
-    cout << "Card5 is now a \"" << card1.getType() << "\" card." << "\n" << "\n"; // testing getter
+    card5.setType("Diplomacy");
+    cout << "Card5 is now a \"" << card5.getType() << "\" card." << "\n" << "\n";
```

Comment:
minor, this applies **everywhere**, u dont need to create more streams, u can just say "\n\n" instead of "\n" << "\n". 
remember, strings are immutable, which means, each time we create a string, we take a new space in memory. its more efficient to keep it as one strig. 

This change should be applied everywhere:)